core/tests/protractor_utils/AdminPage.js

```diff
@@ -46,6 +46,21 @@ var AdminPage = function() {
   var unfinishedOffJobIDClassName = (
     '.protractor-test-unfinished-one-off-jobs-id');
 
+  // Adding a new community reviewer.
+  var addReviewerName = element(by.css('.protractor-add-reviewer-name'));
+  var selectReviewerRole = element(by.css('.protractor-select-reviewer-role'));
+  var addReviewerFormSubmit = element(by.css(
+    '.protractor-add-reviewer-form-submit'));
```

Comment:
Could you change these to start with `.protractor-test-` instead of just `.protractor-`? That helps people search through the code for these kinds of classes